### PR TITLE
Probably fixes the two html UI bugs

### PR DESCRIPTION
**Pull request by clusterfack**
Fixes the bug where the map isn't replaced properly so you never see the UI because the .js isn't sent to you, I should hope, considering I'm telling it to try resending resources each time you open the thing again.

Fixes the bug where the map is the same between rounds when it shouldn't be, by removing the chance of collision between maps. I know I said that before but I did it wrong that time and the js was never modified so it never could have worked.

Also added some whitespace to give readability to some of the headers.

### CODE REVIEW

File: code/modules/html_interface/map/adv_sec/advcamera.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 $(window).on("onUpdateContent", function(){
 	$("#textbased").html("<table><colgroup><col id=\"name\" span = \"2\" style=\"min-width: 40px; width: 30%;\" /><col id=\"pos\" style=\"min-width:150px; width: 30%;\" /></colgroup><thead><tr><td><h3>Name</h3></td><td><h3>&nbsp;</h3></td><td><h3>Position</h3></td></tr></thead><tbody id=\"textbased-tbody\"></tbody></table>");
 
-	$("#uiMap").append("<img src=\"minimap_" + z + ".png\" id=\"uiMapImage\" width=\"256\" height=\"256\" unselectable=\"on\"/><div id=\"uiMapContent\" unselectable=\"on\"></div>");
+	$("#uiMap").append("<img src=\"" + mapname + z + ".png\" id=\"uiMapImage\" width=\"256\" height=\"256\" unselectable=\"on\"/><div id=\"uiMapContent\" unselectable=\"on\"></div>");
```
Comment (clusterfack) [inline code/modules/html_interface/map/adv_sec/advcamera.js]: This line is a new variable dependent upon the map and thus the minimap render will be different between maps

File: code/modules/html_interface/map/adv_sec/adv_camera.dm [hunk 2]
```diff
@@ -64,11 +66,24 @@ var/global/datum/interactive_map/camera/adv_camera = new
 		var/datum/html_interface/hi
 
 		if (!src.interfaces["[z]"])
-			src.interfaces["[z]"] = new/datum/html_interface/nanotrasen(src, "Security Cameras", 900, 800, "[MAPHEADER] </script><script type=\"text/javascript\">var z = [z]; var tile_size = [world.icon_size]; var maxx = [world.maxx]; var maxy = [world.maxy];</script><script type=\"text/javascript\" src=\"advcamera.js\"></script>")
+			src.interfaces["[z]"] = new/datum/html_interface/nanotrasen(src, "Security Cameras", 900, 800, \
+			"[MAPHEADER] </script><script type=\"text/javascript\">\
+			var mapname = [getMinimapShort()]; \
```
Comment (clusterfack) [inline code/modules/html_interface/map/adv_sec/adv_camera.dm]: Here is where that map variable is defined

File: code/modules/html_interface/map/adv_sec/adv_camera.dm [hunk 3]
```diff
@@ -55,6 +55,8 @@ var/global/datum/interactive_map/camera/adv_camera = new
 /datum/interactive_map/camera/show(mob/mob, z, datum/html_interface/currui)
 	z = text2num(z)
 	if (!z) z = mob.z
+	send_resources(mob.client)
```
Comment (clusterfack) [inline code/modules/html_interface/map/adv_sec/adv_camera.dm]: Both of these send resources, we 99.99% know the issue with the map not replacing is a .js not being sent issue and browse_rsc isn't very expensive if you already have the files anyway.

And if it doesn't fix it we'll have another whole angle to consider.

File: code/modules/html_interface/map/adv_sec/adv_camera.dm [hunk 4]
```diff
@@ -55,6 +55,8 @@ var/global/datum/interactive_map/camera/adv_camera = new
 /datum/interactive_map/camera/show(mob/mob, z, datum/html_interface/currui)
 	z = text2num(z)
 	if (!z) z = mob.z
+	sendResources(mob.client)
```
Comment (clusterfack) [inline code/modules/html_interface/map/adv_sec/adv_camera.dm]: Both of these send resources, we 99.99% know the issue with the map not replacing is a .js not being sent issue and browse_rsc isn't very expensive if you already have the files anyway.

And if it doesn't fix it we'll have another whole angle to consider.

**Comment by clusterfack** [comment]
I figured I would have to at least fix these two bugs before I convert the central atmos computer to html interface. And @autismspider was crying about them so I figured they must be pretty old by now.

**Comment by Probe1** [comment]
hooray

**Comment by Exxion** [comment]
Holy shit there are so many issues this should fix
I'll go compile a list

**Comment by Exxion** [comment]
If I'm not mistaken and this works, it should fix:
#8975
#5625
#8961
#8646
And #7999

**Comment by ComicIronic** [comment]
Testing with this merge

**Comment by Exxion** [comment]
This doesn't fix the caching error that caused the maps to not work at all for some people, though it does change the error message
Rather than complaining about clearAll or add not being defined, it now complains about mapname and maxy

**Pull request merged**